client/blocks/visit-site/index.jsx

```diff
@@ -22,7 +22,7 @@ export default function VisitSite( { siteSlug } ) {
 		return null;
 	}
 
-	const siteLink = <a href={ site.URL }>{ site.name.trim() }</a>;
+	const siteLink = <a href={ site.URL }>{ siteSlug }</a>;
```

Comment:
Site slug for some Jetpack sites can include a subdirectory, it's then like `example.com::subsite`. We should replace the `::` with `/` before displaying. We do that at a few places in Calypso already. There can be multiple `::`'s if the path is deeper.

The use case for this "or visit example.com" link in login was that sometimes customers would copy a Calypso link from their browser URL bar:
```
https://wordpress.com/stats/example.com
```
and send that link to their friends like it was a link to their site, i.e., to the frontend. Then obviously the receiver is very confused when they see a login form for the admin interface instead.

Reply:
Aye! The feature is still pretty important for that.

Fixed the issue of subdirectories in https://github.com/Automattic/wp-calypso/pull/70228/commits/4c39ded6c8b06b4fa58ba47edb5e1ed6723d56df